src/engine/strat_engine/engine.rs

```diff
@@ -65,7 +66,7 @@ pub struct StratEngine {
     // In memory filesystem for passing keys to Clevis.
     // See GitHub issue: https://github.com/stratis-storage/project/issues/212.
     #[allow(dead_code)]
-    key_fs: MemoryFilesystem,
+    fs: MemoryFilesystem,
```

Comment:
Are you able to drop comment and also dead code annotation above?

Reply:
The dead code annotation is still required but I've modified the comment!